### PR TITLE
Lottery rules, decorations and horizontal scroll fix

### PR DESCRIPTION
**Pull request by Chef-Cheems**
Preview: https://how-to-play.netlify.app/lottery

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 
-# 10+ nodes balanced, US/EU
-REACT_APP_NODE_1 = "https://bsc-dataseed1.ninicoin.io"
-
-# 10+ nodes balanced, US/EU
-REACT_APP_NODE_2 = "https://bsc-dataseed1.defibit.io"
-
-# 10+ nodes balanced in each region, global
-REACT_APP_NODE_3 = "https://bsc-dataseed.binance.org"
+REACT_APP_CHAIN_ID = "97"
```
Comment (ChefHutch) [inline .env.development]: You can use a `.env.development.local` file & gitIgnore it to manage this locally without risking pushing it

File: src/views/LotteryV2/components/WinningNumbers.tsx [hunk 2]
```diff
@@ -1,63 +1,20 @@
 import React from 'react'
-import styled from 'styled-components'
-import { Flex, Text, Box } from '@pancakeswap/uikit'
+import { Flex } from '@pancakeswap/uikit'
 import { parseRetreivedNumber } from '../helpers'
-import { PinkBall, LilacBall, TealBall, AquaBall, GreenBall, YellowBall } from '../svgs'
-
-const BallTextWrapper = styled.div`
```
Comment (Chef-Cheems) [inline src/views/LotteryV2/components/WinningNumbers.tsx:7]: Balls.tsx now contains ready-to-use balls.

File: src/views/LotteryV2/components/HowToPlay.tsx [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+      <StepCardInner height="220px">
+        <MatchExampleContainer>
+          <Box />
+          <ExampleBalls />
+          <Text lineHeight="72px" textAlign="right" color="secondary" bold mr="20px">
+            A
+          </Text>
+          <MatchExampleA width={exampleWidth} height="46px" isDark={isDark} />
+          <Text lineHeight="72px" textAlign="right" color="secondary" bold mr="20px">
+            B
```
Comment (ChefHutch) [inline src/views/LotteryV2/components/HowToPlay.tsx]: Reckon we need to localise with `t`? For other scripts maybe?

File: src/views/LotteryV2/components/WinningNumbers.tsx (hunk 2 above)
Comment (ChefHutch) [reply src/views/LotteryV2/components/WinningNumbers.tsx:7]: Absolutely dreamy

File: src/views/LotteryV2/index.tsx [hunk 4]
```diff
@@ -100,10 +94,8 @@ const LotteryV2 = () => {
         </Flex>
       </PageSection>
 
-      <PageSection hasCurvedDivider={false} index={0}>
-        <Flex>
-          <img src="/images/lottery/tombola.png" alt="tombola bunny" height="auto" width="240px" />
-        </Flex>
+      <PageSection hasCurvedDivider={false} index={1} containerProps={{ style: { marginTop: '-32px' } }}>
```
Comment (ChefHutch) [inline src/views/LotteryV2/index.tsx]: I think you can remove the `marginTop: -32px` here - it'll also make the spacing more standardised between the sections.

File: src/views/LotteryV2/index.tsx [hunk 5]
```diff
@@ -76,15 +71,14 @@ const LotteryV2 = () => {
           </Flex>
           <NextDrawCard />
         </Flex>
-      </TicketsSection>
-      <PageSection background={CHECK_PRIZES_BG} hasCurvedDivider={false} index={2}>
+      </PageSection>
+      <PageSection background={CHECK_PRIZES_BG} hasCurvedDivider={false} index={4}>
```
Comment (ChefHutch) [inline src/views/LotteryV2/index.tsx]: Maybe the indexes can all be 0 or 1 with the exception of the top `Hero` section, which should be 1 higher?

File: src/views/LotteryV2/index.tsx [hunk 6]
```diff
@@ -48,14 +42,15 @@ const LotteryV2 = () => {
...
+      <PageSection
+        containerProps={{ style: { marginTop: '-30px' } }}
         background={GET_TICKETS_BG}
-        hasCurvedDivider={false}
-        index={3}
+        concaveDivider
+        curvePosition="top"
+        index={4}
       >
         <Flex alignItems="center" justifyContent="center" flexDirection="column">
```
Comment (ChefHutch) [inline src/views/LotteryV2/index.tsx]: Let's add a bit of padding (top) to this? It gets v tight to the PageSection above, particularly on mobile.

**Review by ChefHutch — COMMENTED** [review]
Fab - couple of Qs & noting the stuff we discussed about the PageSection!

File: src/views/LotteryV2/index.tsx (hunk 4 above)
Comment (Chef-Cheems) [reply src/views/LotteryV2/index.tsx]: Nah, this is actually needed still cause without it there is a huge gap between this and previous section

File: src/views/LotteryV2/index.tsx (hunk 5 above)
Comment (Chef-Cheems) [reply src/views/LotteryV2/index.tsx]: True. Actually they all can be 2 except Hero which needs to be lower, i.e. 1. Updated

**Pull request merged**